modules/qna/src/backend/api.ts

```diff
@@ -41,7 +41,7 @@ export default async (bp: typeof sdk, botScopedStorage: Map<string, QnaStorage>)
     }
   })
 
-  router.get('/question/:id', async (req, res) => {
+  router.get('/questions/:id', async (req, res) => {
     try {
```

Comment:
its convention to name endpoints this way. `questions/` is a collection. `question` would mean its a singleton and theres no other "instances"

Reply:
this is debatable, you are getting a single question from the endpoint, so it should be in singular. For example, if you get qna/questions, i would expect to receive multiple questions. But if I get qua/question, i would expect a single item.

qna/questions would make sense to get a list of all questions, while get/question or put/question makes sense when you deal with only one item. 

I guess the correct way would be qna/questions/question/1 to get a single item

Reply:
Look at any api design, best practices, guides, even the official specification. This is the way to do it. If we create funky rules, it will be harder to understand as it doesn't follow the convention.